cmd/ent/main.go

```diff
@@ -117,6 +136,91 @@ func main() {
 	}
 }
 
+func runMigrateV2ToV3Cmd(c *cli.Context) error {
+	if c.Args().Len() != 2 {
+		return xerrors.Errorf("not enough args, need state root to migrate and height of state")
+	}
+	cleanUp, err := cpuProfile(c)
+	if err != nil {
+		return err
+	}
+	defer cleanUp()
+
+	log := lib.NewMigrationLogger(os.Stdout)
+
+	stateRootInRaw, err := cid.Decode(c.Args().First())
+	if err != nil {
+		return err
+	}
+	hRaw, err := strconv.Atoi(c.Args().Get(1))
+	if err != nil {
+		return err
+	}
+	height := abi.ChainEpoch(int64(hRaw))
+	chn := lib.Chain{}
+
+	// Migrate State
+	store, err := chn.LoadCborStore(c.Context)
+	if err != nil {
+		return err
+	}
+	stateRootIn, err := loadStateRoot(c.Context, store, stateRootInRaw)
+	if err != nil {
+		return err
+	}
+	cfg := migration9.Config{
+		MaxWorkers:        8,
+		JobQueueSize:      100,
+		ResultQueueSize:   10,
```

Comment:
I suggest both of these 10x higher. A "job" of a single account actor is very quick, and a thread will do many of them before context switching, so a few workers can easily drain the queue before the producer catches up.